src/App.js

```diff
@@ -1,25 +1,93 @@
-import logo from './logo.svg';
-import './App.css';
+import "./App.css";
+import { useEffect, useState } from "react";
+import { Selector } from "./components/Selector";
+import { CalculatorWindow } from "./components/CalculatorWindow";
+import { stateTable } from "./stateData";
+import { ComparisonOptions } from "./apps/RatioComparer/ComparisonOptions";
+import Player from "./apps/Player/Index";
+import TopBar from "./components/TopBar";
+import About from "./components/About";
+import { playerStateData } from "./apps/Player/playerStateData";
+import BottomBar from "./components/BottomBar";
 
 function App() {
+  const [stateData, setStateData] = useState(stateTable);
```

Comment:
React `useState` isn't a very good solution for app-wide state like this; I see that you've already basically implemented the [reducer state management approach](https://dev.to/tusharkashyap63/everything-about-reducers-395f) so I'd definitely look into using something like [redux](https://redux.js.org/) (or the less boilerplate-intensive [redux-toolkit](https://redux-toolkit.js.org/)). There are a few advantages:

* it allows you to select which parts of state your components actually care about and then automatically memoizes so that your component will only re-render when the relevant parts of state change. `useState` doesn't let you do this and means that your entire page will re-render every time some part of state changes.
* it allows you to use hooks to retrieve data from app state directly from the components that require it - so you can  just call `useSelector` and `useDispatch` in the components that need to access/update state (respectively) without having to manually pass `stateData` and `setStateData` all the way down through props to every component that needs it (you could also get this functionality with React's `CreateContext` and `useContext`)
* it helps with debugging - the redux devtools give you a view of which updates to state happened in which order which makes a lot of issues way easier to diagnose. (On the flip side it's best not to rely on this too much if you're going into a professional career as you will not always have this utility available).